examples/kubernetes/cross_account_mount/iam-policy-examples/node-deamonset-iam-policy-example.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "elasticfilesystem:ClientMount",
+        "elasticfilesystem:ClientRootAccess",
+        "elasticfilesystem:ClientWrite",
+        "elasticfilesystem:DescribeMountTargets"
```

Comment:
why does node need permission to DescribeMountTArgets again?  the controller will do it at CreateVolume time so that node just gets an IP right. (i can check the code but I got lost again, sorry.)

Reply:
`efs-utils` independently tries to resolve mount target as a fail over in case the `az` passed in as mount option is no longer available.